test/files/pos/cycle-jsoup/Test_2.scala

```diff
@@ -0,0 +1,5 @@
+object Test {
```

Comment:
`-Ybreak-cycles` is no-longer needed in 2.13.x since #6653.

That option is still needed for other test cases, when I tried to remove its implementation I got:

```
partest --update-check \
  /Users/jz/code/scala/test/files/pos/t4744/Bar.scala \
  /Users/jz/code/scala/test/files/pos/cycle/X_2.scala \
  /Users/jz/code/scala/test/files/neg/cycle-bounds.scala \
  /Users/jz/code/scala/test/files/neg/t1224.scala

```